FSJ_django20_project/FSJ/models_award.py

```diff
@@ -0,0 +1,33 @@
+#from django.db import models
+import uuid
+from django.utils.translation import gettext_lazy as _
+from django.db import models
+import datetime
+
+class Award(models.Model):
+	FIRST = 1
+	SECOND = 2
+	THIRD = 3
+	FOURTH = 4
+	FIFTH = 5
+	YEAR_CHOICES = (
```

Comment:
Why are we redefining the exact same choices as in Student instead of accessing Student.YEAR_COICES?

Reply:
Not sure if a model should be accessing attributes from another model so I defined it here as well. However, this is kind of a placeholder anyway because this will be likely be replaced when multiple years of study can be selected